Keas.Mvc/Controllers/Api/DocumentController.cs

```diff
@@ -103,17 +114,19 @@ public async Task<IActionResult> Create([FromBody] Document document)
                 return NotFound();
             }
 
-            var envelope = await _documentSigningService.SendTemplate(document.Person.Email, document.Person.Name, document.TemplateId);
+            var envelope = await _documentSigningService.SendTemplate(document);
 
             var newDocument = new Document
             {
                 Name = document.Name,
                 PersonId = document.PersonId,
                 TeamId = document.TeamId,
                 EnvelopeId = envelope.EnvelopeId,
-                TemplateId = document.TemplateId,
+                TemplateId = document.TemplateId,
+                AccountId = document.AccountId,
+                ApiBasePath = document.ApiBasePath,
                 Active = true,
-                Status = "sent",
+                Status = "Sent",
```

Comment:
Does this match what docusign sends back via their API?  I'd like it to match whatever capitalization that is.

Reply:
I believe it does. But I'll double-check...

Reply:
It looks like the `Completed` status comes back capitalized, so assume it would be the same for `Sent`.